include/pg_shard.h

```diff
@@ -77,6 +77,9 @@ typedef struct DistributedPlan
 } DistributedPlan;
 
 
+#define INVALID_SHARD_ID (-1)
+typedef int64 ShardId;
```

Comment:
Was there a strong need for this change? I think it looks cleaner, but it will make it harder to integrate.